samples/tfm_integration/psa_level_1/README.rst

```diff
@@ -25,6 +25,27 @@ multi-thread application.
 .. _PSA Certified Level 1:
   https://www.psacertified.org/security-certification/psa-certified-level-1/
 
+Key Files
+*********
+
+``psa_crypto.c``
+================
+
+Demonstrates the following workflow:
+
+- Generate a permanent persistent key (prime256v1 or ecdsa-with-SHA256)
```

Comment:
"Permanent persistent key" gives a possible miscommunication with a trait of Key Identifiers in PSA. Key Identifiers are explained as "permanent names for persistent keys". Also used in this scope it seems like a double statement. I think  "persistent key" should suffice

See: https://armmbed.github.io/mbed-crypto/html/overview/functionality.html#key-identifiers

I think this needs to change to the the SECG naming "secp256r1" instead of the ANSI x9.62 "prime256v1"
Both legacy mbed TLS APIs and PSA Crypto APIs use the SECG naming. 

I don't understand what "or ecdsa-with-SHA256" means in this context. There is a possibility that this statement gives a miscommunication about key usage flags that can be set with the key. 

E.g: 
https://armmbed.github.io/mbed-crypto/html/api/keys/policy.html#c.PSA_KEY_USAGE_SIGN_MESSAGE
https://armmbed.github.io/mbed-crypto/html/api/keys/policy.html#c.PSA_KEY_USAGE_VERIFY_MESSAGE

Reply:
I've updated both code and docs to reflect SECG naming convention. The wording `ecdsa-with-SHA256` indicate usage of the key as you said. I've updated the description as `Generate a persistent key: secp256r1 (usage: ecdsa-with-SHA256)`. Is this better?